src/api/meetings.js

```diff
@@ -31,15 +31,17 @@ export const fetchMeetings = (token, startDate, endDate) => {
     .then(response => response.body)
 }
 
-export const editMeeting = (token, meeting, room) => agent
-  .put(`${apiBaseUrl}/room/${room.email}/meeting/${meeting.id}`)
-  .set('x-access-token', token)
-  .send({
-    title: meeting.title,
-    start: meeting.start,
-    end: meeting.end,
-  })
-  .then(response => response.body)
+export const editMeeting = (token, meeting, roomEmail) => {
```

Comment:
ok, i'll fix that.  i think it makes more sense to pass in the object so we have access to more properties just in case.